templates/vsc/ts/notification-express/src/notification/notification.ts

```diff
@@ -0,0 +1,831 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Did you change anything to the original implementation in SDK? If not, I will just skip this file :)

Reply:
The NotificationBot constructor slightly changed.
And Removed NotificationTargetType enum.